cmd/loadtest/loadtest.go

```diff
@@ -295,6 +306,7 @@ func completeLoadTest(ctx context.Context, c *ethclient.Client, rpc *ethrpc.Clie
 		log.Error().Err(err).Msg("There was an issue waiting for all transactions to be mined")
 	}
 	if len(loadTestResults) == 0 {
+		log.Info().Msg("CallOnly mode enabled - blocks aren't mined")
```

Comment:
I don't think that's true all the time. Sometimes transactions are not observed because of a problem with the RPC or something else? It would maybe worth adding another if branch that checks if the user specified `call-only` and return `nil` here?

Reply:
Woah, good catch. I have 100% forgot to add that if-statement 😂 

Updated. :)